code/datums/components/foldable.dm

```diff
@@ -112,3 +116,9 @@
 				fold_component.the_briefcase = null
 			src.thingInside = null
 		..()
+	idcomputercase
+		icon_state = "hopcase"
+		item_state = "hopcase"
+	idcomputercaseC
+		icon_state = "hopcaseC"
+		item_state = "hopcaseC"
```

Comment:
Don't need both of these. If you want both (not against it), rename to something not coupled to the ID computer (`blue_stripe` for one was suggested, find some way of describing the other).

Reply:
changed "hopcase" to "blue_stripe" and "hopcaseC" to "blue_green_stripe"